svcs/amigo/resources/private/info.tmpl.html

```diff
@@ -46,8 +46,8 @@ <h1 class="text-center mt-toppage mb-2"><i class="fa fa-info-circle " aria-hidde
                                     <li> Installation instructions for all clients: <a href="https://www.wireguard.com/install/">https://www.wireguard.com/install/</a></li>
 
                                     <p class="p-2 mt-2 dark-color mb-0" style="font-weight: bold; color:red;">User Scripts</p>
-                                    <li> Install Wireguard script : <a href="https://gitlab.com/-/snippets/2102002/raw/master/install_wireguard.sh">install_wireguard.sh</a> </li>
-                                    <li> Connect Event script: <a href="https://gitlab.com/-/snippets/2102000/raw/master/connectwireguard.py">connect_event.py</a> </li>
+                                    <li> Install Wireguard script : <a href="https://raw.githubusercontent.com/Mymaqn/wireguardhaaukins/main/install_wireguard.sh">install_wireguard.sh</a> </li>
+                                    <li> Connect Event script: <a href="https://raw.githubusercontent.com/Mymaqn/wireguardhaaukins/main/connectwireguard.py">connect_event.py</a> </li>
```

Comment:
There is no difference between the content of the files. What's the catch here ?